hadoop-ozone/dist/src/main/compose/ozone-om-ha/test_disabled.sh

```diff
@@ -1,41 +0,0 @@
-#!/usr/bin/env bash
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
```

Comment:
These files are removed, I think this docker-compose is for testing for ratis log purge usecase.

Reply:
Yes, and it's named as `test_disabled.sh` because it didn't work. As I wrote in the description, I suggested creating a simplified (but working) test and fix the *purge use-case* and *restart use-case* in a separated jira, following the pattern introduced by Attila for the upgrade test.

> ozone-ha acceptance tests are turned off long time ago (it included some life-cycle test to start/stop services. In this patch I simplified the ha cluster and added simple smoketest.

> Later we can restore the lifecycle tests (start/stop) but I would prefer to use a generic approach for all the clusters.

But I am open to do anything with the tests (I can revert the changes, but in that case the change won't be covered), jut to commit this patch after 37 days and unblock the release.

Just let me know what do you suggest.

Reply:
I am okay with it, if there is a cleaner approach to do and if it can be added in the new Jira.
cc @hanishakoneru, any comments on removing this?

Reply:
I think the tests which include restarts / failures can be separated and handled similar to the upgrade test. I think we can keep a simple om-ha (or ozone-ha) test which tests the functionality and collect all the lifecycle related tests (restart, kill dn/om services) in a separated place.

We need more tests there not only for HA but for normal servers, IMHO.

Reply:
@elek sorry for jumping into the discussion late. I am working on fixing the robot tests for HA functionality (the old tests covering the restarts and failovers). 
Instead of removing those files altogether, can create another dir for the new docker compose - _ozone-ha-basic_ or something. Or rename the old compose dir to something else. 

As for changing the life-cycle test to start/stop services, I am open to discuss a better approach. But I guess we can keep that for a different thread.

Thanks @bharatviswa504 for bringing it to my notice.